packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -215,16 +227,33 @@ export const SchemaDrivenAnnotationEditor = (
               </Alert>
             )}
           <Form
+            validator={validator}
             className="AnnotationEditorForm"
             liveValidate={liveValidate}
             noHtml5Validate={true}
-            ArrayFieldTemplate={CustomArrayFieldTemplate}
-            ObjectFieldTemplate={CustomObjectFieldTemplate}
-            FieldTemplate={CustomDefaultTemplate}
-            ref={formRefFromParent ?? formRef}
+            experimental_defaultFormStateBehavior={{
+              emptyObjectFields: 'skipDefaults',
+            }}
```

Comment:
This snippet ensures that `default`/`const` fields (which yield "derived annotations") are not pre-filled in the form, so they are not automatically hard-coded and override the derived value.

Reply:
That makes sense, we don't want to automatically override the derived value on edit.  Does the user see the derived value in the form somehow (does it set the input placeholder value for example)?